xml/storage_iscsi.xml

```diff
@@ -1298,13 +1298,13 @@ o- / ............................ [...]
     </listitem>
    </itemizedlist>
    <para>
-    To familiarize yourself with functionality of targetcli, set up a local
+    To familiarize yourself with the functionality of targetcli, set up a local
     image file as a software target using the <command>create</command> command:
    </para>
 <screen>/backstores/fileio create test-disc <replaceable>/alt</replaceable>/test.img 1G</screen>
    <para>
-    This creates the 1GB <filename>test.img</filename> image in the specified
-    location (in this case <filename>/alt</filename>). Run
+    This creates a 1&nbsp;GB <filename>test.img</filename> image in the
+    specified location (in this case <filename>/alt</filename>). Run
```

Comment:
there should be a space between 1 and GB

Reply:
Unless I misunderstand, that is what this change does -- its purpose is to insert a nonbreaking space between the digit and the units...

Reply:
oh yes, it does :)